pkg/storaged/stratis-panel.jsx

```diff
@@ -119,6 +119,12 @@ export function create_stratis_pool(client) {
                           value: name,
                           validate: name => validate_pool_name(client, null, name)
                       }),
+            CheckBoxes("managed", "",
+                       {
+                           fields: [
+                               { tag: "on", title: _("Manage filesystem sizes") }
```

Comment:
I wonder if this is self explanatory enough, I can't find anything in the RHEL docs about it

Reply:
Yes, I agree. I made up the "Managed filesystem sizes" terminology.  The Stratis term is "non-overprovisioning".  We should at least add a (i) bubble with some explanation, or similar.